person.rb

```diff
@@ -22,7 +22,7 @@ def correct_name
   end
 
   def add_rental(rental)
-    @rental << rental
+    @rentals << rental
     rental.person = self
   end
 
```

Comment:
- [ ] I am getting an error while creating a `rental` because in the `book` class you initialized `rentals` but you don't have it in the attributes they should be
`attr_accessor :title, :author, :rentals`